shopinvader_wishlist/models/product_set.py

```diff
@@ -22,17 +26,30 @@ class ProductSet(models.Model):
     )
 
     def get_line_by_product(self, product_id=None, invader_variant_id=None):
-        if not product_id and not invader_variant_id:
+        # Backward compat
+        _logger.info(
+            "DEPRECATED `get_line_by_product`. Use `get_lines_by_product`"
+        )
+        if product_id:
+            product_id = [product_id]
+        if invader_variant_id:
+            invader_variant_id = [invader_variant_id]
+        return self.get_lines_by_product(
+            product_ids=product_id, invader_variant_ids=invader_variant_id
+        )
+
+    def get_lines_by_product(self, product_ids=None, invader_variant_ids=None):
```

Comment:
Since you get a list of product ids as an input, shouldn't the method be named `get_lines_by_products` ?